doc/tutorial_rsmxval.rst

```diff
@@ -0,0 +1,82 @@
+
+ASAP Example
+~~~~~~~~~~~~
+We are going to use the same example from 2012 Kaggle competition on automated essay scoring that we used for the :ref:`rsmtool tutorial <tutorial>`.
+
+Extract features
+~~~~~~~~~~~~~~~~
+We are using the same features for this data as described in the :ref:`rsmtool tutorial <tutorial>`.
+
+Create a configuration file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The next step is to create an :ref:`experiment configuration file <config_file_rsmxval>` in ``.json`` format.
+
+.. _asap_config_rsmxval:
+
+.. literalinclude:: ../examples/rsmxval/config_rsmxval.json
+    :language: javascript
+    :linenos:
+
+Let's take a look at the options in our configuration file.
+
+- **Line 2**: We define an experiment ID used to identify the files produced as part of this experiment.
+- **Line 3**: We also provide a description which will be included in the various reports. 
+- **Line 4**: We list the path to our training file with the feature values and human scores.  For this tutorial, we used ``.csv`` format, but several other :ref:`input file formats <input_file_format>` are also supported.
+- **Line 5**: This field indicates the number of cross-validation folds we want to use. If this field is not specified, ``rsmxval`` uses 5-fold cross-validation by default.
+- **Line 6**: This field indicates that the human (reference) scores in our ``.csv`` file are located in a column named ``score``.
+- **Line 7**: This field indicates that the unique IDs for the responses in the ``.csv`` file are located in a column named ``ID``.
+- **Line 8**: We choose to use a linear regression model to combine the feature values into a score.
+- **Lines 9-10**: These fields indicate that the lowest score on the scoring scale is a 1 and the highest score is a 6. This information is usually part of the rubric used by human graders.
+- **Line 11**: This field indicates that scores from a second set of human graders are also available (useful for comparing the agreement between human-machine scores to the agreement between two sets of humans) and are located in the ``score2`` column in the training ``.csv`` file.
+- **Line 12**: Next, we indicate that we would like to use the :ref:`scaled scores <score_postprocessing>` for all our evaluation analyses at each step.
+
+Documentation for all of the available configuration options is available :ref:`here <config_file_rsmxval>`.
+
+.. note:: You can also use our nifty capability to :ref:`automatically generate <autogen_configuration>` ``rsmxval`` configuration files rather than creating them manually.
+
+
+Run the experiment
+~~~~~~~~~~~~~~~~~~
+Now that we have our input file and our configuration file, we can use the :ref:`rsmxval <usage_rsmxval>` command-line script to run our evaluation experiment.
+
+.. code-block:: bash
+
+    $ cd examples/rsmxval
+    $ rsmxval config_rsmxval.json output
+
+This should produce output like::
```

Comment:
I know @desilinguist filed an issue to deal with the `pandas` `FutureWarning`s, but I see a lot of them when I run this example. I see this output too, but it is drowned out by all the other stuff. Should this be happening?

Reply:
You see it when you just run this command?! Weird, I don’t see this at all when I run it. I am using Python 3.9 in my environment.

Reply:
Yeah, it's strange. I have Python 3.9 too:
```
❯ mamba list "\bpython\b"
# packages in environment at /Users/mmulholland/Documents/rsmtool_dev/rsmdev:
#
# Name                    Version                   Build  Channel
python                    3.9.10          h1dd9edd_2_cpython    /
python-dateutil           2.8.2              pyhd8ed1ab_0    /
```

And this in my `pandas`:
```
❯ mamba list pandas
# packages in environment at /Users/mmulholland/Documents/rsmtool_dev/rsmdev:
#
# Name                    Version                   Build  Channel
pandas                    1.4.1            py39h4d6be9b_0    /
```

Reply:
Very weird. I see no warnings at all.

Reply:
[log.txt](https://github.com/EducationalTestingService/rsmtool/files/8120117/log.txt)

Reply:
Turned out my `pandas` and `numpy` were too new. Downgrading resulted in no warnings. This is basically expected.